src/app/components/Slideshow/sliderComponents/Slider.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from 'react';
+import { SliderLeftArrow, SliderRightArrow } from './Arrows';
+import SliderIndicator from './Indicators';
+import Slide from './Slides';
+
+import '../Slider.scss';
+
+export default class Slider extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeIndex: 0,
+      isStopped: false
+    };
+  }
+
+  componentDidMount = () => {
+    this.activate();
+  };
+
+  componentWillUnmount = () => {
+    this.deactivate();
+  };
+
+  activate = () => {
+    this.interval = setInterval(this.goToNextSlide, 3000);
+  };
+
+  deactivate = () => {
+    clearInterval(this.interval);
+  };
+
+  isStoppedCheck = () => {
+    const { isStopped } = this.state;
+
+    if (!isStopped) {
+      this.setState({ isStopped: true });
+    }
+  };
+
+  handleClickIndicator = id => {
+    this.goToSlide(id);
+    this.isStoppedCheck();
+  };
+
+  handleClickLeft = () => {
+    this.goToPrevSlide();
+    this.isStoppedCheck();
+  };
+
+  handleClickRight = () => {
+    this.goToNextSlide();
+    this.isStoppedCheck();
+  };
+
+  goToSlide = index => {
+    this.setState({ activeIndex: index });
+  };
+
+  goToPrevSlide = e => {
+    if (e) {
+      e.preventDefault();
+    }
+    const { activeIndex } = this.state;
+    const { slides } = this.props;
+
+    let index = activeIndex;
+    const slidesLength = slides.length;
+
+    if (index < 1) {
+      index = slidesLength;
+    }
+
+    --index;
+
+    this.setState({ activeIndex: index });
+  };
+
+  goToNextSlide = e => {
+    if (e) {
+      e.preventDefault();
+    }
+    const { activeIndex } = this.state;
+    const { slides } = this.props;
+
+    let index = activeIndex;
+    const slidesLength = slides.length - 1;
+
+    if (index === slidesLength) {
+      index = -1;
+    }
+
+    ++index;
+
+    this.setState({ activeIndex: index });
+  };
+
+  render() {
+    const { isStopped, activeIndex } = this.state;
+    const { slides } = this.props;
+    const isLink = true;
+
+    return (
+      <div className="slider" onMouseEnter={this.deactivate} onMouseLeave={isStopped ? null : this.activate}>
+        <SliderLeftArrow onClick={() => this.handleClickLeft()} />
+
+        <ul className="slider__slides">
+          {slides.map((slide, id) => (
+            <Slide key={slide.id} index={id} activeIndex={activeIndex} slide={slide} isLink={isLink} />
```

Comment:
if attributes are more than two it should be on new line

Reply:
prettier changes it while committing